pennylane/transforms/batch_transform.py

```diff
@@ -64,7 +65,19 @@ def map_batch_transform(
     >>> dev = qml.device("default.qubit", wires=2)
     >>> fn(qml.execute(tapes, dev, qml.gradients.param_shift))
     [array(0.99500417), array(0.8150893)]
+
+    .. warning::
+        qml.transforms.map_batch_transform is deprecated and will be removed in a future release.
+        Instead, a transform can be applied directly to a batch of tapes. See :func:`~.pennylane.transform` for more details.

```

Comment:
Very minor, but could we move this up to above the example so you don't have to scroll down on the documentation page to see it?

Reply:
I thought the same, but the requirement was to create an expandable section specifically, and it seems this is the highest possible point